_posts/2019-11-18-experience-agile-2019-part-2.md

```diff
@@ -0,0 +1,221 @@
+
+While growing, how are you going to handle the measurement of progress?  
+The most important measurement of progress is… **a working product**. 
+
+So stop asking for progress reports! Rather, create an environment for autonomy and alignment.  
+Give the teams autonomy to create the solution, while still providing the vision and alignment. 
+Tell them what to do, but let them figure out the solution. 
+
+The image below describes it perfectly.
+
+<img alt="leadership-spotify-culture" src="{{ '/img/2019-11-18-experience-agile-2019-part-2/leadership-spotify-culture.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 750px;">
+
+The agile transformation is a challenge, but there's hopeful news.  
+It's a long journey, but it is possible to achieve.
+
+> Every change, no matter how big or small, starts with the same word... YOU!
+
+## Building a Customer Value Engine for a more Successful Company, by Mario Moreira - written by Astrid Legrand
+
+### Intro
+**Building a Customer Value Engine for a more Successful Company** aims to give enterprise leaders tools and advice to maximize customer value with a high level of agility in their companies. 
+ 
+Some methods explained during this workshop can also be easily used at a project level like, for instance, the five Rs model that helps to visualize the enterprise idea pipeline. 
+
+This tool allows to visualize ideas and treat them from the moment the idea is evoked until the moment it is realized.  
+This process consists of five stages, Record – Reveal – Refine – Realize and Release, which corresponds to a **break down** of the idea into a piece of work.
+
+<img alt="5R Model 1" src="{{ '/img/2019-11-18-experience-agile-2019-part-2/5R-model-1.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 750px;">
+
+#### Recording an idea
+At this stage of the process, when a new idea arises, it is documented. 
+No decision has been taken on it so far, we do not know yet if this idea will be useful, if it will be accepted, developed, if it is a priority or if there are sufficient resources to work on it. 
+
+During this first stage, the idea is simply recorded and documented. 
+In an agile environment, not much time should be invested in this stage and particularly in the documentation because we don't know if this idea will go further through the process.  
+This stage should then allow to understand the idea, to determine who the users and beneficiaries are and to quickly estimate the costs linked to this idea.
+
+#### Revealing the idea
+During this stage of the process, the idea is added to the pool of ideas based on its value and priority. 
+This is the moment when it is discussed and challenged among the stakeholders.
+
+The idea is refined in order to decide if the team will continue to work on it. 
+Even if the idea is great, it doesn't mean that the idea will be developed as constraints and dependencies may exist.
```

Comment:
it doesn't mean that the idea will
to
it doesn't mean that it will
to avoid repetition